lib/google/gax/grpc.rb

```diff
@@ -72,22 +74,23 @@ module Grpc
       # @return A gRPC client stub.
       def create_stub(service_path,
                       port,
-                      chan_creds: nil,
                       channel: nil,
+                      chan_creds: nil,
                       updater_proc: nil,
                       scopes: nil)
         address = "#{service_path}:#{port}"
-        if channel.nil?
-          chan_creds = GRPC::Core::ChannelCredentials.new if chan_creds.nil?
+        if channel
+          yield(address, nil, channel_override: channel)
+        elsif chan_creds
```

Comment:
I see. The reason for this change is because in toolkit if we consolidate the updater_proc and the channel_creds into a single parameter. For the user to be able to customize both a chan_creds and updater proc, they would have to compose their chan_creds themselves. If we leave the behavior as such. Their composed chan_creds would get overwritten by the default updater_proc.